Reference/Config/tinyMceConfig/index-v7.md

```diff
@@ -0,0 +1,173 @@
+---
+versionFrom: 7.0.0
+versionTo: 8.0.0
+---
+
+# tinyMceConfig
+
+Here you will find documentation relating to the options you can add/modify in the  tinyMceConfig.config file. The configuration you set here will be used by each TinyMce instance as it is initialized in Umbraco.
+
+## commands
+
+Inside of the `<commands>` node you will find multiple `<command>` nodes. Each one of these nodes defines a icon/button that can appear on the formatting bar inside of Umbraco and the command that is triggered when it is clicked.
+
+```xml
+<command>
+  <umbracoAlias>code</umbracoAlias>
+  <icon>images/editor/code.gif</icon>
+  <tinyMceCommand value="" userInterface="false" frontendCommand="code">code</tinyMceCommand>
+  <priority>1</priority>
+</command>
+```
+
+`umbracoAlias` defines a unique alias within Umbraco for the command. This alias should not contain any spaces.
+
+`icon` defines the path to an image file to be used on the formatting toolbar. This image should be 16px x 16px in size.
+
+`tinyMceCommand` defines the tinyMceCommand properties.
+
+To further break this down, the `value` attribute is usually an empty value as most commands perform formatting tasks as opposed to returning values. The `userInterface` attribute takes a boolean value indicating whether or not the command has an additional UI component to display e.g. a new dialog to  assist with inserting images. `frontendCommand` defines the name of the command to execute. This value usually matches the value of the `tinyMceCommand` node.
+
+`priority` defines the sort order for the commands and should be sequentially incremented for each new command.
+
+## plugins
+
+Inside of the `<plugins>` node you can find one or more `<plugin>` nodes. Each one of these nodes defines a plugin which extends the functionality of TinyMce. Below you will find the standard plugins configured in a clean install of Umbraco v6.x.
+
+```xml
+<plugins>
+    <plugin loadOnFrontend="true">paste</plugin>
+    <plugin loadOnFrontend="true">inlinepopups</plugin>
+    <plugin loadOnFrontend="true">noneditable</plugin>
+    <plugin loadOnFrontend="true">table</plugin>
+    <plugin loadOnFrontend="false">umbracomacro</plugin>
+    <plugin loadOnFrontend="false">umbracoimg</plugin>
+    <plugin loadOnFrontend="false">advlink</plugin>
+    <plugin loadOnFrontend="false">umbracocss</plugin>
+    <plugin loadOnFrontend="false">umbracoembed</plugin>
+    <plugin loadOnFrontend="false">spellchecker</plugin>
+</plugins>
+```
+
+## validElements
+
+The `validElements` node defines which elements will remain in the edited text when the TinyMce Rich Text Editor (RTE) saves. You can use this to limit the returned HTML to a subset.
+
+This option contains a comma separated list of element conversion chunks. Each chunk contains information about how one element and its attributes should be treated.
+
+Here is an example of the standard valid elements that come with a clean install of Umbraco v6.x.
```

Comment:
According to the YAML tags on this article, it applies to all Umbraco 7 versions.
Would it make sense to take an example from a v7 site instead here? 🤔

Reply:
I haven't updated the V7 page, it's been referring to V6 for over a year!